cmd/scribe/scribe.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	goflag "flag"
+	"os"
+
+	scribecmd "github.com/backube/scribe/pkg/cmd"
+
+	"github.com/spf13/pflag"
+	utilflag "k8s.io/component-base/cli/flag"
+	"k8s.io/component-base/logs"
+	"k8s.io/klog/v2"
+
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+)
+
+var (
+	scribeVersion = "0.0.0"
+)
+
+// this function is copied from oc/cmd/oc/oc.go
+func injectLoglevelFlag(flags *pflag.FlagSet) {
+	from := goflag.CommandLine
+	if flag := from.Lookup("v"); flag != nil {
+		level := flag.Value.(*klog.Level)
+		levelPtr := (*int32)(level)
+		flags.Int32Var(levelPtr, "loglevel", 0, "Set the level of log output (0-10)")
```

Comment:
Do we need both of these?
```
      --loglevel int32                 Set the level of log output (0-10)
      --v int32                        Set the level of log output (0-10)
```

Reply:
probably not, will update

Reply:
fixed, have `-v --version` (bc that's defaulted w/ cobra afaict) then `--loglevel` for the verbosity - fixed the double verbosity set and also the ambiguous `-v` for version & loglevel